apps/web/components/ActionCategoryCard/ActionCategoryCard.tsx

```diff
@@ -141,17 +143,34 @@ const Component = forwardRef<HTMLElement, ActionCategoryCardProps>(
             flexDirection="row"
           >
             <Box>
-              <Button
-                {...(cta.buttonType ?? { variant: cta.variant })}
-                size={cta.size}
-                onClick={cta.onClick}
-                disabled={cta.disabled}
-                icon={cta.icon}
-                iconType={cta.iconType}
-                nowrap
-              >
-                {cta.label}
-              </Button>
+              {cta.href ? (
+                <LinkV2 href={cta.href} newTab={true}>
+                  <Button
+                    {...(cta.buttonType ?? { variant: cta.variant })}
+                    size={cta.size}
+                    fluid
+                    disabled={cta.disabled}
+                    icon={cta.icon}
+                    iconType={cta.iconType}
+                    nowrap
+                  >
+                    {cta.label}
+                  </Button>
+                </LinkV2>
+              ) : (
+                <Button
+                  {...(cta.buttonType ?? { variant: cta.variant })}
+                  size={cta.size}
+                  fluid
+                  onClick={cta.onClick && cta.onClick}
```

Comment:
I have a feeling that this could instead just be `onClick={cta.onClick}`

Reply:
Correct, removed. I incorrectly assumed a runtime error if onClick wasn't provided.